setting-up-multifactor-authentication.md

```diff
@@ -52,11 +52,12 @@ MFA for both the UI and the API. If you go to the _edit settings_ page again, in
 will see a dropdown menu with three options:
 
 - **Disabled**: disables MFA. Please delete rubygems.org account from your authenticator app after disabling.
-- **UI only**: sign in from browser, updating MFA levels and resetting password will require OTP code. These are referred to as UI operations.
 - **UI and gem signin**: UI operations and `gem signin` will require OTP code.
 - **UI and API**: UI operations, `gem signin`, `push`, `owner --add` and `owner --remove` will require OTP code.
 
-Note: If you are on the **UI only** or **UI and gem signin** authentication level,
+UI Only was previously a valid MFA level, however, it has been removed and only accounts that were previously at that level will still see it in the dropdown.
```

Comment:
I imagine the blog post as being linked on `removed`

Reply:
Sure, it would need to be created in the blog repo here: https://github.com/rubygems/rubygems.github.io. Would you like to write up a draft since you worked on this?

Clarifying on my original question, do you have a preference on the merge order? ie. merging this before the blog post has been posted and adding the link after or waiting until the blog has been merged and then merging this?

Reply:
Sorry, I undercommunicated. I meant that I think the blog post linking could be a new pull request after this PR is merged. As for writing the blog post, I could do it, but I don't yet understand why UI only is insecure. Is it just that the CLI usage is more common?